tests/pytests/unit/modules/test_status.py

```diff
@@ -455,6 +455,35 @@ class MockData:
     return m
 
 
+def test_status_diskusage_not_exist():
+    with patch("os.path.exists", return_value=False):
+        assert status.diskusage("/foo/bar/not-existing/") == {}
+
+
+def test_status_diskusage_valid_output():
```

Comment:
Probably just skip this on Windows since Windows has its own `status` module

Reply:
suggestion added